pkg/util/metric/v2/metrics.go

```diff
@@ -223,10 +224,15 @@ func initShardingMetrics() {
 	registry.MustRegister(ReplicaFreezeCNCountGauge)
 }
 
-func getDurationBuckets() []float64 {
-	return append(prometheus.ExponentialBuckets(0.00001, 2, 30), math.MaxFloat64)
-}
+var (
+	minDuration    = float64(time.Nanosecond*100) / float64(time.Second)
+	maxDuration    = float64(time.Minute*10) / float64(time.Second)
```

Comment:
I'm afraid 10 minutes is too short for the upper bound; how about one hour to allow for unusually prolonged tasks?

Reply:
OK